src/test/suite/lib/ggshield-api.test.ts

```diff
@@ -0,0 +1,53 @@
+import { GGShieldConfiguration } from "../../../lib/ggshield-configuration";
+import * as statusBar from "../../../gitguardian-interface/gitguardian-status-bar";
+import * as simple from "simple-mock";
+import { diagnosticCollection, scanFile } from "../../../lib/ggshield-api";
+import * as runGGShield from "../../../lib/run-ggshield";
+import path = require("path");
+import { Uri } from "vscode";
+import assert = require("assert");
+import { scanResultsWithIncident } from "../extension.test";
```

Comment:
I think we could move these constants like all possible scan results, standardized errors, etc... To a separate file. WDYT?